### ISSUE TITLE
Fix S2934 FN: Support unsigned right shift operator (>>>)

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6296]
### Description

FN for S2934 (GenericReadonlyFieldPropertyAssignment).

Using C#11 unsigned right shift operator (>>>), the rule is not raising an issue.

### Repro steps

```csharp
 interface IPoint
    {
        int X { get; set; }
        int Y { get; set; }
    }

    class PointManager<T> where T : IPoint
    {
        readonly T point;  // this could be a struct

        public void MovePointVertically(int newX)
        {
            point.X >>>= 4; // FN
        }
    }

```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
